pkg/api/types/global.go

```diff
@@ -0,0 +1,45 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package types
+
+// GlobalOptions is the global options
+type GlobalOptions struct {
```

Comment:
Could you consider splitting this to a separate PR?

Reply:
Also this looks a duplicate of the TOML config struct
https://github.com/containerd/nerdctl/blob/89f10d8b168c0b227182922497d0667624218d3b/cmd/nerdctl/main.go#L137-L152